Sources/BowEffects/Data/Resource.swift

```diff
@@ -0,0 +1,103 @@
+import Bow
+
+public final class ForResource {}
```

Comment:
I think we should go adding in the same PR the API docs. It is easier done at that moment :)

Reply:
That would be ideal, yes, but this is prone to changes before merging it to master, so I prefer to make sure everything is stable before adding the API reference. But definitely, it must be documented before merging it with master.